### PR TITLE
Ported allies05a to lua

### PR DESCRIPTION
**Pull request by abcdefg30**
I split the lua script into two parts:
One for the mission itself and the other one for the production of the "AI".

### CODE REVIEW

**Comment by Mailaender** [comment]
You can rebase now as https://github.com/OpenRA/OpenRA/pull/7407 fixed the CI build again.

**Comment by Mailaender** [comment]
:+1: works well.

File: mods/ra/maps/allies-05a/allies05a.lua [hunk 1]
```diff
@@ -0,0 +1,276 @@
...
+InitTriggers = function()
+	Trigger.OnInfiltrated(Warfactory, function()
+		Trigger.ClearAll(Spy)
+		greece.MarkCompletedObjective(infWarfactory)
+		WarfactoryInfiltrated()
+	end)
+
+	Trigger.OnInfiltrated(Prison, function()
+		Trigger.ClearAll(Spy)
+		MissInfiltrated()
```
Comment (obrakmann) [inline mods/ra/maps/allies-05a/allies05a.lua]: I'd delay this for 1-2 seconds for effect.

**Comment by obrakmann** [comment]
Very nice mission as usual. :+1: when you add that small delay

**Comment by abcdefg30** [comment]
Updated.

**Comment by obrakmann** [comment]
erp, some more suggestions: https://github.com/abcdefg30/OpenRA/pull/3

**Comment by abcdefg30** [comment]
Updated again.

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog/_compare/39e8c742813014abb95915ce00aac7ebb2a03942%5E...39e8c742813014abb95915ce00aac7ebb2a03942)

**Pull request merged**